components/service/glean/docs/events.md

```diff
@@ -21,13 +26,6 @@ Each entry in the `events` array is an array with the following items:
 
 - [2]: `name`: The name of the event, as definded in the `metrics.yaml` file.
 
-- [3]: `objectId`: An arbitrary string representing the id of the object on
-  which the event occurred. The application is responsible for providing the id
-  to glean when events are recorded.
-
-- [4]: `value` (optional): A application-provided value providing additional
-  context for the event.  Values will never exceed 80 characters.
-
-- [5]: `extra` (optional): A mapping of strings to strings providing additional
+- [3]: `extra` (optional): A mapping of strings to strings providing additional
   data about the event. Both the keys and values in this map will never exceed
```

Comment:
That's for fixing this, I forgot to do it in my PR :( This needs to slightly change too. `keys` in `extra` are enforced to be 40 chars at most at build time, while values are now limited to 100 characters.